### ISSUE TITLE
Always opened dropdown

### ISSUE THREAD

**Issue by nereauweb** [jackocnr/intl-tel-input#1171]
Would it be possible to have the dropdown always opened and active, with current script or slightly modifying it?
Setting in css display: block !important does not work, as it does not initialize the dropdown, and as consequence selection does not work.
I tried to call the _showDropdown and _initDropdownListeners functions, but that does not seem to work.
My actual solution is to create a click event after page load, but that does not solve my need, as the dropdown gets then closed.
Suggestions on how to do this would be appreciated.

Thanks.

**Comment by jackocnr** [comment]
It's not officially supported, but you could just call the `_showDropdown` function. You'd also want to call disable `_bindDropdownListeners` as well, else all of your keyboard keys would be captured by the dropdown, and clicking anywhere on the page would close the dropdown. So it would look something like this:

```js
var iti = intlTelInput(input);
iti._bindDropdownListeners = function() {};
iti._showDropdown();
```

But as I say this is not officially supported and could break with any update.

**Comment by nereauweb** [comment]
Sorry for late feedback:
it works well as per your suggestion, but only with the non-minimized js file. In fact I already tried something similar, your suggestion didn't seem to work, so I decided to try to alter the js and included the extended version - at that point it worked without any internal modification. I guess that in the minified js the function names gets altered.
Thank you for support, very nice plugin by the way! 
Cheers
